Tests/BackendIntegrationTests/StoreKitIntegrationTests.swift

```diff
@@ -11,15 +11,9 @@ import RevenueCat
 import StoreKitTest
 import XCTest
 
-class StoreKit2IntegrationTests: StoreKit1IntegrationTests {
-
-    override class var sk2Enabled: Bool { return true }
-
-}
-
-class StoreKit1IntegrationTests: BaseBackendIntegrationTests {
+class BaseStoreKitIntegrationTests: BaseBackendIntegrationTests {
```

Comment:
Extracting this base class with no tests so `BaseTrialOrIntroEligibilityCheckerIntegrationTests` can inherit from it.